crates.io wgpu-native 0.3.1 fails to build

```
   Compiling wgpu-native v0.3.1
error[E0433]: failed to resolve: use of undeclared type or module `ptr`
   --> C:\Users\Layl\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-native-0.3.1\src\instance.rs:117:47
    |
117 |             instance.create_surface_from_hwnd(ptr::null_mut(), h.hwnd),
    |                                               ^^^ use of undeclared type or module `ptr`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
error: Could not compile `wgpu-native`.

To learn more, run the command again with --verbose.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -59,6 +59,8 @@ script:
   - cargo test
   - (cd wgpu-native && cargo check --features local,glutin)
   # TODO: Temporarily only test building the gl backend, properly test when it is usable from C
+  - if [[ $TRAVIS_OS_NAME == "windows" ]]; then (cd wgpu-native && cargo check --features gfx-backend-dx11); fi
+  - if [[ $TRAVIS_OS_NAME == "windows" ]]; then (cd wgpu-native && cargo check --features gfx-backend-dx12); fi
   - if [[ $TRAVIS_OS_NAME == "linux" ]]; then cargo check --release; fi
   - if [[ $TRAVIS_RUST_VERSION == "nightly" ]]; then cargo +nightly install cbindgen; fi
   - if [[ $TRAVIS_RUST_VERSION == "nightly" ]] && [[ $TRAVIS_OS_NAME == "windows" ]]; then

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,8 @@
 # Change Log
 
+## v0.3.3 (22-08-2019)
+  - fixed instance creation on Windows
+
 ## v0.3.1 (21-08-2019)
   - fixed pipeline barriers that aren't transitions
 

--- a/wgpu-native/Cargo.toml
+++ b/wgpu-native/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "wgpu-native"
-version = "0.3.2"
+version = "0.3.3"
 authors = [
 	"Dzmitry Malyshau <kvark@mozilla.com>",
 	"Joshua Groves <josh@joshgroves.com>",

--- a/wgpu-native/Cargo.toml
+++ b/wgpu-native/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "wgpu-native"
-version = "0.3.1"
+version = "0.3.2"
 authors = [
 	"Dzmitry Malyshau <kvark@mozilla.com>",
 	"Joshua Groves <josh@joshgroves.com>",

--- a/wgpu-native/src/instance.rs
+++ b/wgpu-native/src/instance.rs
@@ -115,7 +115,7 @@ pub fn wgpu_instance_create_surface(
         #[cfg(all(windows, feature = "gfx-backend-vulkan"))]
         raw_window_handle::RawWindowHandle::Windows(h) =>
             instance.create_surface_from_hwnd(std::ptr::null_mut(), h.hwnd),
-        #[cfg(all(windows, feature = "gfx-backend-dx12"))]
+        #[cfg(all(windows, feature = "gfx-backend-dx11"))]
         raw_window_handle::RawWindowHandle::Windows(h) =>
             instance.create_surface_from_hwnd(h.hwnd),
         #[cfg(all(windows, feature = "gfx-backend-dx12"))]

--- a/wgpu-native/src/instance.rs
+++ b/wgpu-native/src/instance.rs
@@ -114,7 +114,7 @@ pub fn wgpu_instance_create_surface(
             instance.create_surface_from_wayland(h.display, h.surface),
         #[cfg(all(windows, feature = "gfx-backend-vulkan"))]
         raw_window_handle::RawWindowHandle::Windows(h) =>
-            instance.create_surface_from_hwnd(ptr::null_mut(), h.hwnd),
+            instance.create_surface_from_hwnd(std::ptr::null_mut(), h.hwnd),
         #[cfg(all(windows, feature = "gfx-backend-dx12"))]
         raw_window_handle::RawWindowHandle::Windows(h) =>
             instance.create_surface_from_hwnd(h.hwnd),

